book/src/kubernetes-changelog.md

```diff
@@ -3,6 +3,28 @@
 This page summarizes major CSI changes made in each Kubernetes release. For
 details on individual features, visit the [Features section](features.md).
 
+## Kubernetes 1.23
+* Beta
+    * Add Fsgroup Mount
+* Alpha
+    * Recover from Expansion Failure
+    * Auto delete PVCs created by StatefulSet
+    * Honor PV Reclaim Policy
+    * RBD CSI Migration
+    * Portworx CSI migration
+
+## Kubernetes 1.22
+* GA
+    * Windows CSI (CSI-Proxy API v1)
+* Alpha
```

Comment:
These are also available in alpha:
https://github.com/kubernetes/enhancements/issues/2317
https://github.com/kubernetes/enhancements/issues/1495

Reply:
added